integration-tests/features/steps/vulnerablity-analysis.py

```diff
@@ -0,0 +1,196 @@
+"""Tests for API endpoints that performs v2 Vunerablity analysis route."""
+import requests
+import time
+import logging
+from behave import then, when
+from urllib.parse import urljoin
+from src.parsing import parse_token_clause
+from component_analysis_v2_batchcall import get_packages_from_json
+logger = logging.getLogger()
+
+
+def vulnerability_analysis_batch_call_url(context):
+    """Construct url for call."""
+    return urljoin(context.core_v2_api_url,
+                   '/api/v2/vulnerability-analysis')
+
+
+def get_token_call_url(context):
+    """Construct url for get-token call."""
+    return urljoin(context.core_v2_api_url,
+                   '/api/v2/get-token')
+
+
+def perform_Vuln_batch_call(context, use_user_key, packages_file):
+    """Perform Batch Call for vulnerability-analysis."""
```

Comment:
how many reqs are you going to send in total for this?

Reply:
Somewhere around 10 - 11 including all the cases 
Only positive cases 5